tools/github-event-processor/Azure.Sdk.Tools.GitHubEventProcessor/GitHubEventClient.cs

```diff
@@ -1321,23 +1321,18 @@ public virtual RulesConfiguration LoadRulesConfiguration(string rulesConfigLocat
             return rulesConfiguration;
         }
 
-        /// <summary>
-        /// Class to hold the response from the AI Label Service query
-        /// </summary>
-        public class LabelResponse
+        public virtual async Task<IssueTriageOutput> QueryAILabelService(IssueEventGitHubPayload issueEventPayload)
         {
-            public string[] Labels { get; set; }
-        }
+            //Empty response for failures
+            IssueTriageOutput empty = new IssueTriageOutput { Labels = [], Suggestion = null, Solution = null };
```

Comment:
We should make this a static method on `IssueTriageOutput` so that we can do  `IssueTriageOutput.Empty` here.